src/components/CardView/CardView.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from "react";
+
+import "./CardView.scss";
+
+class CardView extends Component {
+  state = {
+    isFlipped: false,
+    isGuessed: false
+  };
+
+  handleFlippingCard = () => {
+    const changeState = () => {
```

Comment:
I'm no expect but this seems like not the correct way. Please check out https://reactjs.org/docs/handling-events.html